### ISSUE TITLE
Windows scrolls on press left/right arrows

### ISSUE THREAD

**Issue by tboby** [Pseudo-Corp/Synergism-Issue-Tracker#18]
**Describe the bug**
Playing on Kongregate, with a resolution of 1366x768, the "change tab" shortcuts of left and right arrow cause the screen to scroll left and right. This occurs even in "cinematic" mode.

A workaround is to catch the keydown event in the game's iframe
```
window.addEventListener("keydown", function(e) {
    // space and arrow keys
    if([32, 37, 38, 39, 40].indexOf(e.keyCode) > -1) {
        e.preventDefault();
    }
}, false);
```

**To Reproduce**
1. Open game with a resolution below around 1400
2. Optionally enable Cinematic mode 
3. Click on the game to focus on it
4. Press the right arrow key

**Logs:**
```
js-sec.indexww.com/ht/p/186080-85072989693750.js:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
synergism?haref=HP_TGTM_synergism:34 The Performance Observer MUST have at least one valid entryType in its entryTypes attribute.
__nr_require.2 @ synergism?haref=HP_TGTM_synergism:34
op.js:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
ima3.js:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
get.s-onetag.com/a016d5b9-e250-4037-a13c-d8384aa31127/tag.min.js:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
videojs_javascripts-b034c206e93a3eafa1c60b144ee641b04775a70c58eaf85da127ebef8a5bc95c.js:537 Uncaught ReferenceError: google is not defined
    at b.ima (videojs_javascripts-b034c206e93a3eafa1c60b144ee641b04775a70c58eaf85da127ebef8a5bc95c.js:537)
    at IMAVideoBumper.createAdPlayer (holodeck_javascripts-493c6610a2a86eda29a10d68336005a69c7ba1958da9315bd63fae89504db403.js:873)
    at IMAVideoBumper.showPreroll (holodeck_javascripts-493c6610a2a86eda29a10d68336005a69c7ba1958da9315bd63fae89504db403.js:873)
    at IMAVideoBumper.loadGame (holodeck_javascripts-493c6610a2a86eda29a10d68336005a69c7ba1958da9315bd63fae89504db403.js:871)
    at synergism?haref=HP_TGTM_synergism:384
    at HTMLDocument.<anonymous> (sitewide_javascripts-a2ce0782e47ef12c75e8b3fe96041de67a0c0fcfa535adcbe77e342f259818c8.js:801)
    at HTMLDocument.f (sitewide_javascripts-a2ce0782e47ef12c75e8b3fe96041de67a0c0fcfa535adcbe77e342f259818c8.js:273)
    at h (sitewide_javascripts-a2ce0782e47ef12c75e8b3fe96041de67a0c0fcfa535adcbe77e342f259818c8.js:276)
    at HTMLDocument._methodized [as fire] (sitewide_javascripts-a2ce0782e47ef12c75e8b3fe96041de67a0c0fcfa535adcbe77e342f259818c8.js:121)
    at HTMLDocument.a (sitewide_javascripts-a2ce0782e47ef12c75e8b3fe96041de67a0c0fcfa535adcbe77e342f259818c8.js:283)
cdn-kongregate-com.videoplayerhub.com/gallery.js:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
holodeck_javascripts-493c6610a2a86eda29a10d68336005a69c7ba1958da9315bd63fae89504db403.js:587 Unexpected token <
onException @ holodeck_javascripts-493c6610a2a86eda29a10d68336005a69c7ba1958da9315bd63fae89504db403.js:587
js-agent.newrelic.com/nr-1167.min.js:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
s.adroll.com/j/roundtrip.js:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
synergism?haref=HP_TGTM_synergism:1 [DOM] Found 2 elements with non-unique id #comment_user_id: (More info: https://goo.gl/9p2vKq) <input type=​"hidden" name=​"comment[user_id]​" id=​"comment_user_id">​ <input type=​"hidden" name=​"comment[user_id]​" id=​"comment_user_id">​
/site_stats.json:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
ssl.google-analytics.com/__utm.gif?utmwv=4.9.4&utms=1&utmn=1908715794&utmhn=www.kongregate.com&utmcs=UTF-8&utmsr=1366x768&utmsc=24-bit&utmul=en-us&utmje=0&utmfl=-&utmdt=Play%20Synergism%2C%20a%20free%20online%20game%20on%20Kongregate&utmhid=834035412&utmr=-&utmp=%2Fwebplayer%2Finstall%2Fmissing&utmac=UA-16068464-16&utmcc=__utma%3D216929292.644756070.1539976695.1589666426.1589720271.106%3B%2B__utmz%3D216929292.1588455410.101.2.utmcsr%3Dreddit.com%7Cutmccn%3D(referral)%7Cutmcmd%3Dreferral%7Cutmcct%3D%2F%3B&utmu=qACg~:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
Synergism.js:665 1589668565493
Synergism.js:675 Offline progress has allowed you to gain 0 obtainium!
Synergism.js:687 You were offline for 51708.103 seconds!
```
**Expected behavior**
Both inside and outside Cinematic mode, the screen should not scroll.

**Device:**
 - OS: Ubuntu 64-bit
 - OS Version: 18.04
 - Browser: Chrome (stable)
 - Browser Version: Version 74.0.3729.131 (Official Build) (64-bit)
 - Have reproduced on latest Windows 10 + latest Chrome too
 - I have PiHole set up on my network, hence failed advert requests

**Additional Information:**
- The tab does change, it just *also* scroll

**Comment by KhafraDev** [comment]
I believe this issue has been fixed, a fix will be released by v1.009.

**Comment by tboby** [comment]
This appears to have been fixed, thanks!
